src/protocolgame.cpp

```diff
@@ -2432,7 +2431,7 @@ void ProtocolGame::sendCreatureSay(const Creature* creature, SpeakClasses type,
 	msg.addByte(0x00); // "(Traded)" suffix after player name
 
 	// Add level only for players
-	if (const Player* speaker = creature->getPlayer()) {
+	if (auto speaker = creature->getPlayer()) {
```

Comment:
Could this be `const auto`
```suggestion
	if (const auto speaker = creature->getPlayer()) {
```